### PR TITLE
 Add ICLR 2019, 2018

### PR DESCRIPTION
**Pull request by loalm**

### CODE REVIEW

**Comment by loalm** [comment]
All available papers from ICLR with a decision seem to be added now. 2016, 2017 seem unavailable. 2014, 2013 papers have no decisions.

**Comment by JonasRSV** [comment]
Hey, I know that atleast for ICLR 2019  the decision notes does not contain the "decision" field. I started working on fixing so that the is_accept heuristic works for 2019 as well by using the rating field. But I am not sure that that code is correct yet, nor am I sure it works for 2018.

Did you make sure that it works correctly for 2018 and 2019? If not please do that first :)

**Review by JonasRSV — CHANGES_REQUESTED** [review]
See previous comment.

**Comment by loalm** [comment]
@JonasRSV I changed the decision_notes for ICLR 2018 and ICLR 2019 + changed some parts of the accept heuristic. Could you take a look at this? :)

File: src/Tools/open_review_downloader.py [hunk 1]
```diff
@@ -97,15 +116,20 @@ def __is_accepted(self, note):
...
-            # TODO(..): Work on this heuristic
-            if int(note.content['rating'][0]) > 5:
-                return True
-            return False
+
+        #Works for ICLR 2019
+        if "recommendation" in note.content:
+            return "Accept" in note.content["recommendation"]
+
+        # Commented this out, not sure if this is needed anymore
```
Comment (JonasRSV) [inline src/Tools/open_review_downloader.py:127]: You can remove this if it is not used anymore :)

File: src/Tools/open_review_downloader.py [hunk 2]
```diff
@@ -97,15 +116,20 @@ def __is_accepted(self, note):
...
         # If it is from ICLR and it is not a year with decision field its a bit more tedious --
         # Instead in their 'Official Reviews' there is a field called 'rating'
         # The first character of the rating is a number 0-9 if it is larger than 5 it appears to be accepted.
-        if 'rating' in note.content:
-            # TODO(..): Work on this heuristic
-            if int(note.content['rating'][0]) > 5:
-                return True
-            return False
+
+        #Works for ICLR 2019
```
Comment (JonasRSV) [inline src/Tools/open_review_downloader.py:123]: Does it also work for 2018?

File: src/Tools/open_review_downloader.py [hunk 3]
```diff
@@ -59,6 +59,25 @@ class Paper:
...
+    #Seems to work , gives 917 rejected, 502 accepted
+    ScrapeURLs(submission_notes="ICLR.cc/2019/Conference/-/Blind_Submission",
+           decision_notes="ICLR.cc/2019/Conference/-/Paper.*/Meta_Review"),
+
+    #Seems to work, gives 598 rejected, 337 accepted
+    ScrapeURLs(submission_notes="ICLR.cc/2018/Conference/-/Blind_Submission",
+           decision_notes="ICLR.cc/2018/Conference/-/Acceptance_Decision"),
+
+
+    ##### WORKSHOPS
```
Comment (JonasRSV) [inline src/Tools/open_review_downloader.py:72]: Nice!

**Review by JonasRSV — CHANGES_REQUESTED** [review]

**Comment by JonasRSV** [comment]
Looks great! Once you have removed that un-commented code and answered weather it works for 2018 also, ping me again and i'll approve it! :)

**Comment by loalm** [comment]
@JonasRSV Great, I have removed the unused part of the accept heuristic. As for ICLR 2018, the accept heuristic works similarly as with MIDL.

**Review by JonasRSV — APPROVED** [review]

**Comment by JonasRSV** [comment]
Looks good! Feel free to merge :)

**Pull request merged**